code/modules/shuttle/super_cruise/orbital_map_components/orbital_objects/space_station.dm

```diff
@@ -0,0 +1,16 @@
+/datum/orbital_object/z_linked/station
+	name = "Space Station 13"
+	mass = 0
```

Comment:
No mass?

Reply:
No, mass only determines gravitational attraction of other objects towards it